server/testutil/testfs/testfs.go

```diff
@@ -0,0 +1,116 @@
+package testfs
```

Comment:
nit: it would be nice to replace all of the ioutil usages with the equivalent io package functions.

Reply:
Done